configs/config.dev.toml

```diff
@@ -4,18 +4,18 @@ logLevel = "debug"
 
 [broker]
 topicOffsetPath = "offset.txt"
-url = "localhost:8889"
+url = "localhost:8888"
 topicID = 0
 token = "secretToken"
 
 [blockchain]
-depositkey = "5JJvavvETHcGEx7ueoDry8zvcfPenbBCKcUQXXXXXX"
-signidicekey = "5JJvavvETHcGEx7ueoDry8zvcfPenbBCKcUQXXXXXX"
-rsakey = "base64 encoded pem key"
+depositkey = "5Jx1vdKxdmeFbdqFuKMRLanHVy8jgVnSXDAiP3AKheympfCkC6H"
+signidicekey = "5KVV7UwoBYpqV6z5XxrUgfADqQZxT2xC8x5PGg9zLJ7998Qxv8V"
 url = "https://api.daobet.org"
 casinoAccountName = "daocasinoxxx"
 platformAccountName = "platform"
-platformPubKey = "EOS1234"
+rsakey = "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBMlhVanlZTWVnQ1lDczRhUEVzb0hXM0hqLy9iR0ZpUmE5UmR6dDExYWhQNzRhc240ClIybTc4YkttZDVTV1ZLQzhvVDNRd1RDRjVsWWIyOHZLLzVFNmtOVjJPc2htdVZ6WVkxNHhLSDBXbDR5L2R2a2QKd1FJRTVxKzN5RGFneGRPRmwwbjZxVmhmdUdTTjdCZnJvdEZIZGRKZUZlWEk1ZmNwd1BqMnVhaUFHODhnN0tmVgowSGtwSmwxaUQ2R09CUUQ2a0pPLy9KNnlOUmtnNkl3NERTMk55OTJIZ0xmK29QUys2NGZnclRlcW9lNGR1R09GCnFtQkJKUHJRb0lreTNibUJJa3JoQWJnZVJ4NTRTQmxlcDhyQ0diTFNTRjJoVVh0eitwVVRsMVcydGJ0Ulh6YTAKUUwzUUVDUVdFejBHczYxVFVmeUJ1RUZ5cmlsM3gvOWdUZENlZ1FJREFRQUJBb0lCQUFxWU8zTWc4VVo3RTV5ZApqTEQwcU9FYXJtb1ZJdGVIVStzbkJsdERzQnBVdGRPMkhXT1ZValBYYVdiZ0ZXYndJZUNWTzJ0K0tyZnJNQlpJCldvNlRDOUJzcGdMOUtvd1Rhdk9FYms2enNYc1d2MmxUZjlDY2daS2Zzd3VOSmM0ZG5SK2UyaGlRRnJITUxHSDUKVkdDdFk1Ym85ZTZKTUgyTHlqdjlwVCtzWnZsenlmalJPODdTMkpEam9tQVBEY291Qzd1dXFSbDRObk5pNEVObQowVDNDR2JoNGVzMUtMWTVlN3dYUzQvbDZkbGdKemVQZjNuOFdoNkRSLzhObU9OZnJsVXhBd1R3czdTbVdrSEU3CnZKWlp3UGdsUi8vNTVZR1hsWGtWc2F4SGtMNGZMeGNJRzdETEtHR0FSdzZFUVJ6ZGx3ZTJRcEhhQ3V0amNnMXYKUjIyYUc4RUNnWUVBK0EzSW5DSWZOSFlXb3p5YjBwckZpeThjRTd0OGdZWjV1OUNnWW8vMVhTYys3bERES1FCMApZUmRDMjBMelpxbkhaem9GNTNhKzJCVE5yYmxFS3VVYkI5cm1oUlBEa2RxMGs3UUhXV3ZnUitkYVp0ZEtNRUFiCkN1V0drMGNZN2VsYzY0RnQ2dnB1a3M4dVZYeDYrNG9IY3RQeCtaOEQ4a0ZrN0cxNDkrRHl2dmtDZ1lFQTRHeHgKK2UvaDV1dHVBcGpjWDZ6emhsVXJTSUY5UnBvbTF0Y05pVWJBeFQ0UEpJejJJVEEzZEo1UmkzQlRNVCtUV3pNWgpyZi9xVUFVNDRCWGFURmx2eTArNTNmckZOOHdWdE05aXhYelVwTnVuM0tEeTdwVldpeTd3blgvWHJuSVkzbTdPClFxc0ZuOUVRQVU2VFFEcXJRYzBTSUM2dG11MnBGc25JS2FoUVZja0NnWUFGNVozOVNEcGhydVQrNEhQNzM3NE8KZndJMTNlRXAyeDZOdm5ueG5lYUJYbXQ2a3VUdzFwZnRydlhoN0llSXkwUjRTY0xXSzlhQm9JejdNa3BTa3J1Lwp4ZzV3MzNWb3hFQkN1cWJxRTB4enFTWlhCNHMxOW5BdUpDSzg0WlhRSWhvWlp5eEx4OFJWTFNPMjJJeTc2aDRmCk5yYXI3b0hUZ29aNXNHaU56QVU1VVFLQmdDWGdoN2dNVVk4OHl2dHJPejUyblQ5ZDNvbEZPWlNSN2Q0eXU1a2cKdE9aVlhBN3FMNXhZVy9PMXlNT0UrTFdHS21PcEV6dDN4dDhza25YWG1sbHFldURRbXVrcEt1SUk4dHpxWjV3MwpQbGg0cmpNaGd2N2JlNWpRR2xJcXdvY3h3SnJDRHRQWHM0Sk9zYW5XemlsMy9YR0VuL3VJaFVweHRDa2hjMFA4Cm9WcHBBb0dCQUlTeDRsZXQ0MmhSUzErR2phTDFtYi9jbnFDRWF5ZzVrZXpzVFBJRENSdzZCQnE3blhkWElwcTAKR05wTFBGeTFLQlM2WElGdlNVbzhPd3BnUXdnNzRvTy83eTA3K1kwSnVQYUpUTnhmZ0hoSUxxbVN0VGlSMi9UQQpQUnpFaHZOekRXTjhVamZpYUdUK0xwbHRRZExURGJuQ0llYTdTei81eU1HOEdqTHdXdTcyCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0t"
+platformPubKey = "EOS51QLenoFHatEB6WxLh5c7aRYfDKKFNW8c9NT7EMC9u7or2CMVo"
```

Comment:
no need to commit keys....

Reply:
these are not actual keys that are used in production, they're just there for validation to work